src/ducks/demo/AugmentedModal.styl

```diff
@@ -1,32 +1,35 @@
-.AugmentedModalIntent
-    padding 0
+.ContainerIntentSide
+    display flex
+    height 100%
+    padding 0 !important
 
-.AugmentedModalDescription--NoHeader
-    margin-top 0 !important
-
-.AugmentedModalDescription
-    padding 0!important
-    & > div
-        height 100%
-        & > *
-            box-sizing border-box
-
-            &:last-child
-                flex 0 1 35%
-                padding 1.5rem
+.IntentIframeContainer
+    flex 5
 
 .FakeInfos
     position relative
+    flex 2
+    padding 20px
+    background-color #F5F6F7
+    overflow auto
 
 .FakeInfosLinkVendor
 .FakeInfosLinkDrive
     position absolute
-    left 0
-    width 125px
+    left 20px
     height 25px
 
 .FakeInfosLinkVendor
-    top 203px
+    width 140px
+    top 225px
 
 .FakeInfosLinkDrive
-    top 359px
+    width 104px
+    top 380px
+
+.FakeVeePeeHeader
+    background-color #eee
+    display flex
+    justify-content center
+    width calc(100% - 64px) !important
+    padding 10px 32px !important
```

Comment:
Naive question: why do we have to put hardcoded values and !important.

Reply:
we need to override some values ​​concerning the DialogContent and DialogTitle components

Reply:
Do we need to have the exact same visual result as before ?

Reply:
Yes

Reply:
OK then :)